src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -150,9 +150,16 @@ def load_arguments(self):
         if self.arguments_loader:
             cmd_args = self.arguments_loader()
             if self.no_wait_param:
+                if isinstance(self.no_wait_param, six.string_types):
+                    no_wait_param_dest = self.no_wait_param
+                elif callable(self.no_wait_param):
+                    no_wait_param_dest = '_no_wait_dest_for_callable'
```

Comment:
I saw this string is used multiple places, consider extract a constant variable.

Reply:
made a const.